metadata-service/restli-client/src/main/java/com/linkedin/entity/client/EntityClient.java

```diff
@@ -218,4 +223,39 @@ public SearchResult filter(
       int count,
       @Nonnull String actor)
       throws RemoteInvocationException;
+
+  public VersionedAspect getAspect(
+      @Nonnull String urn,
+      @Nonnull String aspect,
+      @Nonnull Long version,
+      @Nonnull String actor)
+      throws RemoteInvocationException;
+
+  public VersionedAspect getAspectOrNull(
+      @Nonnull String urn,
+      @Nonnull String aspect,
+      @Nonnull Long version,
+      @Nonnull String actor) throws RemoteInvocationException;
+
+  public List<EnvelopedAspect> getTimeseriesAspectValues(
```

Comment:
Request for opinion: Thoughts on rebranding this as "getTimeseriesAspects"

Reply:
See answer above ^